### ISSUE TITLE
Add Liam's ROSES PyGMT talk to website

### ISSUE THREAD

**Issue by seisman** [GenericMappingTools/pygmt#613]
**Description of the desired feature**

@liamtoney recently gave an excellent presentation about PyGMT in the [ROSES summer school](https://www.iris.edu/hq/inclass/course/roses). The video of the presentation is now [available on YouTube](https://www.youtube.com/watch?v=SSIGJEe0BIk). 

It's a good tutorial for new beginners. I believe there will be more and more videos like this (either from PyGMT developers or users) in the future. Perhaps we should have a page listing all available learning resources?

**Comment by weiji14** [comment]
> @liamtoney recently gave an excellent presentation about PyGMT in the [ROSES summer school](https://www.iris.edu/hq/inclass/course/roses). The video of the presentation is now [available on YouTube](https://www.youtube.com/watch?v=SSIGJEe0BIk).

Just watching the video now, it's very well done! @liamtoney, do you have a link to the slides/jupyter notebooks somewhere? Really like the fancy powerpoint layer animation showing the postscript layers:

![Postscript GMT map layers](https://user-images.githubusercontent.com/23487320/93530473-bb874500-f991-11ea-9b44-cce6420ffeb0.png)

> It's a good tutorial for new beginners. I believe there will be more and more videos like this (either from PyGMT developers or users) in the future. Perhaps we should have a page listing all available learning resources?

I think this video is simply brilliant :raised_hands:, and deserves to be on the front page! We should also refresh our main README.md to put up some gallery examples (see how GMT.jl does it at https://github.com/GenericMappingTools/GMT.jl/tree/v0.23.0) and tone down on those scary disclaimers.

**Comment by liamtoney** [comment]
Didn't realize it was up on YouTube until now! Thanks for the kind words. It was a lot of fun — happy to share more at our next meeting. The materials (video, notebooks) *should* be available at the "ROSES summer school" link @seisman posted above soon.

In the meantime, the notebooks can be accessed at this GitHub repo: https://github.com/fdannemanndugick/roses2020/tree/master/unit8

**Comment by weiji14** [comment]
FYI the video and jupyter notebooks are now available at https://www.iris.edu/hq/inclass/lesson/728. I've also started #643 to add these to the PyGMT website :smile:

**Comment by weiji14** [comment]
A few months overdue, but @liamtoney's ROSES 2021 Mapping lecture is up at https://www.youtube.com/watch?v=Zvcy7VDuhiw. It's a bit longer this time (1hr 23min) and includes some mention of Cartopy, but I think having the comparison is useful, especially for people more familiar with the Python world than GMT. Note that the video references PyGMT v0.4.0, but still better than the 2020 version that referenced v0.1.2.

So, should we update the front page and/or add this video to https://www.pygmt.org/v0.5.0/overview.html#presentations? We *could* release a PyGMT v0.5.1 to make it more visible to users.

P.S. I'm secretly hoping that @meghanrjones will have a nice 3 min lightning talk prepared for AGU 2021 which we can put up on the front page in December (no pressure :wink:).

**Comment by meghanrjones** [comment]
> A few months overdue, but @liamtoney's [ROSES 2021](https://connect.agu.org/seismology/roses2021/roses2021materials) Mapping lecture is up at https://www.youtube.com/watch?v=Zvcy7VDuhiw. It's a bit longer this time (1hr 23min) and includes some mention of Cartopy, but I think having the comparison is useful, especially for people more familiar with the Python world than GMT. Note that the video references PyGMT v0.4.0, but still better than the 2020 version that referenced v0.1.2.
> 
> So, should we update the front page and/or add this video to https://www.pygmt.org/v0.5.0/overview.html#presentations? We _could_ release a PyGMT v0.5.1 to make it more visible to users.
> 
> P.S. I'm secretly hoping that @meghanrjones will have a nice 3 min lightning talk prepared for AGU 2021 which we can put up on the front page in December (no pressure 😉).

It will be great to include a link to Liam's latest lecture! I love that they have included closed captioning!

Yes, I've registered for AGU and confirmed my 3-minute virtual lightning talk. I am hoping to draft up the presentation earlier than I have done in the past so anyone could provide comments in early December.

**Comment by michaelgrund** [comment]
>  We should also refresh our main README.md to put up some gallery examples (see how GMT.jl does it at https://github.com/GenericMappingTools/GMT.jl/tree/v0.23.0) and tone down on those scary disclaimers.

Should we also add some of the gallery examples in the same breath as @weiji14 mentioned in the original post last year?

**Comment by liamtoney** [comment]
My goal with the 2021 lecture was to provide more of a comparison between PyGMT and cartopy. The lecture has a lot of content overlap with the 2020 lecture, however. And to be frank, the 2021 lecture is not my best work — I think the 2020 lecture was smoother.

It's good that the newer lecture leverages newer PyGMT. But replacing the front page with @meghanrjones lightning talk could be a good change — my screenshot has been hogging the main page for a while!

Thanks, @weiji14.

**Comment by meghanrjones** [comment]
Given the zoom meeting format for the eLightning session last week, I gave a 3-minute overview of PyGMT live rather than using a pre-recorded talk. There was a recording message on the screen, but I have no idea who owns that recording. It would probably be easier to create a separate recording than track that down. The material for the poster are at https://github.com/meghanrjones/agu2021 and I created some last minute slides using keynote 😳 that I could convert to a more open format for sharing. Comments are welcome and I would take thumb's up as an indicator that I should create the recording.
